redaxo/src/core/assets/standard.js

```diff
@@ -732,6 +744,36 @@ jQuery(document).ready(function($) {
     });
 
     document.addEventListener('keydown', handleKeyEvents, true);
+
+    // Dropdown Search
+    $('.rex-js-dropdown-search').on('keyup', function () {
```

Comment:
Ich hatte den Code weiter nach oben geschoben, mit in `rex:ready`. Jetzt ist er doppelt hier drin.

Reply:
Habe ich hiermit https://github.com/redaxo/redaxo/pull/4722/commits/d0018213fef4ef56af320c5ad81c9696bbae040f angepasst und den JS-Code aus dem vorherigen Commit nach oben an deine Stelle in `rex:ready` gezogen und deine Änderungen mit dem Selector `viewRoot.find` und den Verzicht auf die Variable `moduleName` auch übernommen.